CONTRIBUTING.md

```diff
@@ -1,41 +1,50 @@
+First of all, thank you for your interest in Overload! Any contribution is welcome, including:
 
 - Reporting a bug
 - Submitting a fix
 - Proposing new features
-- Improving the code quality
+- Improving code quality
 
 ## We Develop with Github
-We use github to host code, to track issues and feature requests, as well as accept pull requests.
+We use GitHub to host code, track issues and feature requests, and accept pull requests.
 
 ## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
-Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+Pull requests are the best way to propose changes to the codebase (we follow [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests.
 
-1. Fork the repo
-2. Create your branch from `develop` respecting naming conventions
-3. Review your code before submitting (Build and quality check)
-4. Create a pull request to `develop`
+To create a pull request:
+
+1. Fork the repository.
+2. Create your branch from `develop` (following naming conventions).
+3. Review your code before submitting (conduct build and quality checks).
+4. Create a pull request targeting the `develop` branch.
 
 ## Any contributions you make will be under the MIT Software License
-In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+In short, when you submit code changes, your contributions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
 
 ## Report bugs using Github's issues
-We use GitHub issues to track public bugs. Report a bug by opening a new issue it's that easy!
-
-## Use a Consistent Coding Style
-* Interfaces starts by `I`
-* Abstracts starts by `A`
-* Class names: `UpperCamelCase`
-* Public member variables: `lowerCamelCase`
-* Private member variables: `m_lowerCamelCase`
-* Public static variables: `UpperCamelCase`
-* Private static variables: `_CAPS_LOCK_WITH_UNDERSCORES`
-* Function/Method arguments: `p_lowerCamelCase`
-* Function/Method names: `UpperCamelCase`
-* Class member variables are located on file bottom
+We use GitHub issues to track bugs. Report a bug by opening a new issue, it's that easy!
+
+## Coding Conventions
+* Interfaces starts with `I`.
+* Abstracts starts with `A`.
+* Class names: `UpperCamelCase`.
+* Public member variables: `lowerCamelCase`.
+* Private member variables: `m_lowerCamelCase`.
+* Public static variables: `UpperCamelCase`.
+* Private static variables: `_CAPS_LOCK_WITH_UNDERSCORES`.
```

Comment:
I am not really convinced by the naming convention of the static variables with the underscore at the beginning that are usually for reserved variables, also the double underscore naming that is for example used in the Clock class from OvTools but it's not part of the convention so should be corrected soon.

Would suggest to just remove the underscore at the beginning and keeping the Caps lock with underscores

Reply:
I agree, I didn't modify these in this pass, but I guess it's a great time to look into it. Not sure about `UPPER_SNAKE_CASE` for private static variables though. I'd like to suggest the most commonly used `s_lowerCamelCase`. What do you think?

Reply:
Yes let's stick to the Hungarian notation for that!